hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestConfigurationProperties.java

```diff
@@ -122,4 +125,58 @@ public void testGetPropertiesWithPrefixEmptyResult() {
     Assert.assertEquals(0, propsLongPrefix.size());
     Assert.assertEquals(0, propsNonExistingRootPrefix.size());
   }
+
+  @Test
+  public void testWhiteListConstructor() {
```

Comment:
The latest revision looks good to me, but please add unit tests to verify that the properties kept in sync even with deprecation and alternative names. (To verify PropWithListener)